plugins/discourse-cakeday/db/migrate/20251127125226_delete_old_default_values.rb

```diff
@@ -21,6 +21,8 @@ def migration_timestamp(version)
   end
 
   def delete_setting(name, time)
+    return if name.nil? || time.nil?
```

Comment:
name is always non-nil

are there any cases where the timestamp would be missing? this migration follows the other two